bricks/dart_frog_prod_server/hooks/pre_gen.dart

```diff
@@ -64,25 +77,23 @@ Future<void> preGen(
     },
   );
 
-  await reportExternalPathDependencies(
-    projectDirectory,
-    onViolationStart: () {
-      context.logger
-        ..err('All path dependencies must be within the project.')
-        ..err('External path dependencies detected:');
-    },
-    onExternalPathDependency: (dependencyName, dependencyPath) {
-      context.logger.err('  \u{2022} $dependencyName from $dependencyPath');
-    },
-    onViolationEnd: () {
-      exit(1);
-    },
-  );
-
   final customDockerFile = io.File(
     path.join(projectDirectory.path, 'Dockerfile'),
   );
 
+  // Get all the internal path packages
+  final pathDependencies = (await getPathDependencies(projectDirectory))
+      .where((dependencyPath) => !dependencyPath.startsWith('..'))
```

Comment:
I would use the `path` package here. For a file path to be parent it doesn't always start with `..`, consider: `foo/../../`

Reply:
good call, I replaced this elsewhere, was going to replace here, but seems that I forgot. I have changed to use path.